### DISCUSSION TITLE
有提供x86 32位的so吗

### DISCUSSION THREAD

**Discussion by caihuanjian** [Tencent/libpag#2522] [General] [answered]
有提供x86 32位的so吗

**Comment by kevingpqi123** [comment, marked as answer]
现在提供的 SDK 架构如下：
Android 端 SDK 为 aar 文件，支持 armeabi、armv7a、arm64
iOS 端 SDK 为 framework 文件，提供动态库完整版本（arm64、模拟器 x84_64/arm 64）

如果需要其它架构，pag 的源码是开源的，需要自己编译
